### DISCUSSION TITLE
Tiled Upscale & Noise Injection

### DISCUSSION THREAD

**Discussion by action1618** [Acly/krita-ai-diffusion#1440] [Q&A]
Hello everyone, first want to offer thanks. Krita-ai is likely by far my favorite resource or stable diffusion ever, offering the consistency and simplicity of a111 but the power of comfyui and bridging the gaps in the weakness in both. I absolutely love this resource and have been blown away by the results.

Q1. I am curious if there is any way we can emulate a control-net tiled upscale, as in the past this has produced some the highest detail and quality I have personally experienced? In short breaking the target image into smaller regions rendering each one as its own image and then stitching them together. This might be the only feature I might default back to some of my previous tools, but was curious if there is a method internal as you all seem to have though of a lot great resources.

Q.1.5 The other small element I was curious about is some type of re-render with noise injection, essentially a follow up render that helps add more detail into the existing dimensions without up scaling. Relatively common with comyfui. I think you might be able to do this relatively close with a manual layer gradient and transparency but again was curious if anything native exists im overlooking?

**Comment by nabileisa** [comment]
Regarding your first question :
Tiled upscale using controlnet is already available in the plugin , it just has a different name `Unblur` , if you go to the upscale view 
![image](https://github.com/user-attachments/assets/16781166-e767-40b1-952c-329064eed8b3)
 you will find `image guidance` , this control the strength or toggle `controlnet-tile` on and off , it defaults to `Unblur - Medium`
![image](https://github.com/user-attachments/assets/8ac6adca-f0f1-4d50-8c26-4681e4cd60fe)
while the `strength` slider above it controls `denois`
